firefox-ios/firefox-ios-tests/Tests/XCUITests/TodayWidgetTests.swift

```diff
@@ -379,8 +497,12 @@ class TodayWidgetTests: BaseTestCase {
         // Reopen and interact with the Copied Link widget
         tapOnWidget(widgetType: "Copied Link")
         // Handle paste alert
-        mozWaitElementHittable(element: springboard.alerts.buttons["Allow Paste"], timeout: 3)
-        springboard.alerts.buttons["Allow Paste"].tap()
+        if #available(iOS 16, *) {
+            mozWaitElementHittable(element: springboard.alerts.buttons["Allow Paste"], timeout: 3)
+            springboard.alerts.buttons["Allow Paste"].tap()
+        }
+        // mozWaitElementHittable(element: springboard.alerts.buttons["Allow Paste"], timeout: 3)
```

Comment:
Do we need to keep these two lines which are commented?

Reply:
Removed commented lines